cpp/tests/CMakeLists.txt

```diff
@@ -690,10 +690,6 @@ ConfigureTest(STREAM_DICTIONARY_TEST streams/dictionary_test.cpp STREAM_MODE tes
 ConfigureTest(STREAM_FILLING_TEST streams/filling_test.cpp STREAM_MODE testing)
 ConfigureTest(STREAM_GROUPBY_TEST streams/groupby_test.cpp STREAM_MODE testing)
 ConfigureTest(STREAM_HASHING_TEST streams/hash_test.cpp STREAM_MODE testing)
-# Deprecation from 16297 and fixes in 16379 caused this test to be empty This will be reenabled once
-# the deprecated APIs have been replaced in 24.10.
-#
-# ConfigureTest(STREAM_INTEROP_TEST streams/interop_test.cpp STREAM_MODE testing)
```

Comment:
The stream tests are not be re-enabled?

Reply:
No. The other tests that were disabled are tests of using the Arrow C++ library to produce objects and then convert them to cudf objects, or vice versa. While we are removing those APIs, we still support the same functionality via the C Data Interface, so I think reenabling the other tests is important because it gives us confidence that we can in fact still support that use case. This stream test, on the other hand, is just to verify that internally the old API correctly passed streams all the way down. We do need to test the new APIs for this, but I don't see any reason to jump through the same hoops of creating Arrow C++ objects etc, which is what I would have to do to reenable the test above. Instead, we should just write new stream tests for the new APIs that create the C Data Interface objects directly via nanoarrow. I can do that, but didn't want to clutter this PR.